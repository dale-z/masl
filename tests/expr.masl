// 38 // 0;
int a = 1 + 2 - 3 * 4;
double b = a;
bool c = (true && false) || (3 > 2);
char d = 'a';
[char] e = [char]{d};
double f(int a, double b) { return a*b; }
(int, double):double g = f;
double h = g(a, b);

printInt(a);
printChar('\n');
printDouble(b);
printChar('\n');
printBool(c);
printChar('\n');
printChar(d);
printChar('\n');
printChar(e:[0]);
printChar('\n');
printDouble(h);
printChar('\n');
