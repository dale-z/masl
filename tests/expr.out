-9
-9.0
true
a
a
81.0
